lib/Cargo.toml

```diff
@@ -12,3 +12,4 @@ failure = "0.1"
 itertools = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+nom = "5.0.1"
```

Comment:
I think we should just use `major.minor`, so that we don't have to make changes to pick up patch versions.

Reply:
o littty